source/main.cpp

```diff
@@ -226,6 +230,9 @@ int main(int argc, char *argv[])
 	Audio::Quit();
 	GameWindow::Quit();
 
+	// Exit curl.
+	curl_global_cleanup();
```

Comment:
Need to call this in the catch above too if an error occurs

Reply:
done